R/deploymentTarget.R

```diff
@@ -76,13 +77,17 @@ deploymentTargetForApp <- function(appId,
   accountDetails <- findAccount(account, server)
   application <- getApplication(accountDetails$name, accountDetails$server, appId)
 
+  resultAppId <- ifelse(is.null(application$content_id), application$id, application$content_id)
+
   createDeploymentTarget(
```

Comment:
I think it's now worth naming all these arguments too, since otherwise I can't easily tell what you've changed here.

Reply:
Actually, I'll make that change so you can just bring in my code.